tests/integrationv2/providers.py

```diff
@@ -472,21 +472,6 @@ def supports_protocol(cls, protocol, with_cert=None):
 
     @classmethod
     def supports_cipher(cls, cipher, with_curve=None):
-        if "openssl-1.0.2" in get_flag(S2N_PROVIDER_VERSION) and with_curve is not None:
```

Comment:
Is this case still still being tested? Just want to make sure it's not suddenly working without skipping these tests because the curve/cipher isn't being tested anymore for some reason. Then if we add the tests back we'd have to add this exception again.

Reply:
Yes! I checked through the integration test logs and we can see lines like the following
```
 [gw1] [ 36%] PASSED test_happy_path.py::test_s2n_server_happy_path[RSA_4096_SHA512-TLS1.2-P-384-S2N-OpenSSL-ECDHE-RSA-AES128-GCM-SHA256] 
```

Reply:
Nice!